app/_hub/kong-inc/ai-proxy/how-to/_streaming.md

```diff
@@ -0,0 +1,64 @@
+---
+nav_title: Streaming
+title: Configure Streaming with AI Proxy
+---
+
+This guide walks you through setting up the AI Proxy plugin with streaming.
```

Comment:
@ttyS0e Could you explain what streaming is from the perspective of a customer use case?

Reply:
I done it